sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -170,11 +170,21 @@ public int getMaxFrameSize() {
         return handler.getConnectionProperties();
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
     public Mono<AmqpSession> createSession(String sessionName) {
+        return createSession(sessionName, false);
+    }
+
+    /**
+     * Creates a new session with the given session name and set it up for distributed transaction if required.
+     *
+     * @param sessionName Name of the session to be created.
+     * @param distributedTransactionsSupport If this session should support distributed transactions across entities.
+     * @return The AMQP session that was created.
+     *
+     * @see <a href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-transactions-v1.0-os.html#section-coordination">Distributed Transactions</a>
+     */
+    protected Mono<AmqpSession> createSession(String sessionName, boolean distributedTransactionsSupport) {
```

Comment:
You have createSessionOptions but this just uses a boolean

Reply:
Removed this API need.